tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -247,6 +248,50 @@ compute_distance2electron (rpacket_t * packet, storage_model_t * storage)
   return rpacket_get_tau_event (packet) * inverse_ne;
 }
 
+INLINE void
+compute_distance2continuum(rpacket_t * packet, storage_model_t * storage)
+{
+  double chi_boundfree, chi_freefree, chi_electron, chi_continuum, d_continuum;
+
+  	  // Compute the continuum oddities for a real packet
+//	  calculate_chi_bf(packet, storage);
+
+  chi_boundfree = 0.2e-15;	//calculate_chi_bf(packet, storage);
```

Comment:
I think for these values it would be useful to write getters. these can for now produce preset results, but later be linked to atomic data.